docs/sites.yml

```diff
@@ -4001,3 +4001,12 @@
   built_by: Michael Uloth
   built_by_url: "https://www.michaeluloth.com"
   featured: false
+  - title: Gábor Fűzy pianist
```

Comment:
Hi @B3zo0,

Please can you indent the title in lines with other entries. Thank you!

Reply:
yep 😅